qutip/core/superoperator.py

```diff
@@ -52,24 +52,22 @@ def _map_over_compound_operators(f):
     """
     @functools.wraps(f)
     def out(qobj):
-        if isinstance(qobj, QobjEvoBase):
-            return qobj.linear_map(f)
+        if isinstance(qobj, QobjEvo):
+            return qobj.linear_map(f, _skip_check=True)
         if not isinstance(qobj, Qobj):
             raise TypeError("expected a quantum object")
         return f(qobj)
     return out
 
 
-def liouvillian(H, c_ops=[], data_only=False, chi=None):
+def liouvillian(H=None, c_ops=None, data_only=False, chi=None):
     """Assembles the Liouvillian superoperator from a Hamiltonian
-    and a ``list`` of collapse operators. Like liouvillian, but with an
-    experimental implementation which avoids creating extra Qobj instances,
-    which can be advantageous for large systems.
+    and a ``list`` of collapse operators.
 
     Parameters
     ----------
     H : Qobj or QobjEvo
-        System Hamiltonian.
+        System Hamiltonian or Hamiltonian composant of a Liouvillian
```

Comment:
```suggestion
        System Hamiltonian or Hamiltonian composed of a Liouvillian
```

Reply:
No, the Liouvillian is build from the Hamiltonian not the other war around.
If `composant` is confusing (or am I forcing a French word here?), how about `term` or `part`

Reply:
"composant" isn't an English word (though it almost feels like it should be...) - you probably mean "component"